### ISSUE TITLE
Myoclonic tremor

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#5772]
**Preferred term label:**
Myoclonic tremor

**Synonyms**

**Definition (free text, please give PubMed ID)**
A tremor-like cortical myoclonus. The tremulous movements are in fact small, high-frequency myoclonic jerks, induced by posture or action. They can be mild, but can also be more incapacitating. When present in the lower limbs, these may lead to gait disorders and even drop attacks. These tremor-like movements during action can be mistaken for ET, or epileptic seizures. 
PMID:29416935

**Parent term (use hpo.jax.org/app)**
https://hpo.jax.org/app/browse/term/HP:0001336
https://hpo.jax.org/app/browse/term/HP:0001337

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
https://omim.org/entry/618075

**Your nano-attribution (ORCID)**
